cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -144,6 +144,11 @@ function up() {
     fi
     $kubectl label node -l $label node-role.kubernetes.io/worker=''
 
+    if [ "$KUBEVIRT_MULTI_NUMA" == "true" ]; then
```

Comment:
Why do we need this?

Reply:
This is not necessary to run multi NUMA node but it makes it easy to filter nodes which has more than one NUMA node, for example we can implement such function for test purposes:
```go
func GetMultiNUMANodesCount(virtCli kubecli.KubevirtClient, nodeName string) int {
       node, err := virtCli.CoreV1().Nodes().Get(context.Background(), nodeName, k8smetav1.GetOptions{})
       Expect(err).ShouldNot(HaveOccurred())

       numaNodesLabel := "kubevirtci.io/numa_nodes"

       for label, val := range node.GetLabels() {
               if label == numaNodesLabel {
                       numaNodes, err := strconv.Atoi(val)
                       if err != nil { // if label value can't be converted to integer assume only 1 NUMA node
                               return 1
                       }

                       return numaNodes
               }
       }
       return 1
}
```

Reply:
The side effect is that the tests would be very specific to kubevirtci. Or to put it another way, others would need to modify their clusters. I do not object against it as I can see an example when one might have only a subset of nodes with `numa nodes>1` in the cluster and would need to signal it to testsuite.

Reply:
another solution we can use is to install `node-feature-discovery` and rely on `NodeResourceTopology` which exposes [NUMA topology](https://github.com/k8stopologyawareschedwg/noderesourcetopology-api/blob/master/pkg/apis/topology/v1alpha1/types.go#L39) via custom resources.

Reply:
Actually, we might be exposing this already with the node labeller. Let me have a look. But I like what you suggest as that is more of a standard than making our own label.

Reply:
If this go through I'll prepare another PR to add `nfd` to one of the provisioners